source/Engine.cpp

```diff
@@ -449,6 +530,8 @@ void Engine::Wait()
 // Begin the next step of calculations.
 void Engine::Step(bool isActive)
 {
+	static const Color &playerShipSelectedPointerColor = *GameData::Colors().Get("ship target pointer player");
```

Comment:
You use this also only once, why the variable? Also the static is a big no-no.

Reply:
I wasn't sure how many times I'd use it at the time.
Shouldn't the static thing be fine, though?
~~Isn't everything in the namespace static?~~

Reply:
The issue is what you assign to the static: When in the future we'll change it so that plugins can be reloaded without closing the game, this value would still refer to the original one.